ocfweb/tv/templates/tv/tv-base.html

```diff
@@ -0,0 +1,30 @@
+{% load staticfiles %}
+
+<!doctype html>
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>
+      {% if title %}
+      {{ title }} &ndash;
+      {% endif %}
+      Open Computing Facility
+    </title>
+
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,700,400italic,700italic,500,500italic" rel="stylesheet" />
+    <link href="{% static 'scss/site.scss.css' %}" rel="stylesheet" />
+    <link rel="icon" href="{% static 'img/favicon/favicon-16.png' %}" sizes="16x16" />
+    <link rel="icon" href="{% static 'img/favicon/favicon-32.png' %}" sizes="32x32" />
```

Comment:
is this entire head tag copied? can you move this to a partial and include it in both places rather than duplicating?